### ISSUE TITLE
page not found

### ISSUE THREAD

**Issue by AhmedTharwat-AT** [nextauthjs/next-auth#10535]
### What is the improvement or update you wish to see?

https://authjs.dev/guides/basics/refresh-token-rotation  
this page returns 404 error ( not found)

### Is there any context that might help us understand?

https://authjs.dev/guides/basics/refresh-token-rotation  
this page returns 404 error ( not found)

**Comment by ndom91** [comment]
Good catch, although we're not super happy with it - we've added it back for now until we can come up with a better one :pray:
